src/main/java/interface_adaptors/show_history/ShowHistoryFileDataAccess.java

```diff
@@ -26,7 +26,7 @@ public class ShowHistoryFileDataAccess implements ShowHistoryDsGateway {
 
     //Return true if stringdate a is bigger than stringdate b
     public ShowHistoryFileDataAccess() throws IOException {
-    reader = new GatewayReader("D:\\uoft\\2022 fall\\course-project-group-131\\src\\main\\java\\historydatabase\\history.csv");
+    reader = new GatewayReader("C:\\Users\\ken_w\\Desktop\\Uoft 2022-2023\\Fall 2022\\CSC 207 Software design\\Login System\\src\\main\\java\\historydatabase\\history.csv");
```

Comment:
I guess there is a problem here because from the laptop we copied the repository path but when it updated it automatically added the laptop information in front.